client/src/pages/Explorer.js

```diff
@@ -73,47 +98,82 @@ function FavoriteCheckBox() {
 
 const ExplorerPage = () => {
   const classes = useStyles();
+  const [generalData, setGeneralData] = useState({places: [], favorites: []});
+  const citiesUrl = "/api/cities/us/?population=1500000";
+  const userFavoriteCitiesUrl = "/api/users/favoriteCities";
```

Comment:
as per my comment above, we can now fetch cities with a `GET` request to `/api/users/:userId/favorite-cities`

Reply:
Both this URL and `citiesURL` are only used once. There's no point putting them in a variable